src/reducers/searchResultsReducer.js

```diff
@@ -69,36 +69,39 @@ export default function course(state = initialState.SearchResults, {payload, typ
     }
 }
 
-const handleAccountSearchResults = (state, payload, status) =>{
-    let {response}= payload;
-    let {data} = response;
-
+const handleAccountSearchResults = (state, payload, status) => {
+    let { response } = payload;
+    let { data } = response;
+    console.log(data)
+    // you can get page and count
     return {
         ...state,
-        accounts:data.results,
+        count: data.count,
+        page: parseInt(data.page),
```

Comment:
page shouldn't be stored here. You should be tracking 2 pages: the current page for Account results, and the current page for Course results